google-cloud-bigtable/lib/google/cloud/bigtable/table_admin_client.rb

```diff
@@ -227,6 +227,280 @@ def drop_row_range \
+        #   client = Google::Cloud::Bigtable.new(
+        #     client_type: :table,
+        #     instance_id: "instance-id"
+        #   )
+        #
+        #   snapshot = client.snapshot("users-snapshot")
+
+        def snapshot \
+            snapshot_id,
+            cluster_id,
+            options: nil
+          client.get_snapshot(
+            snapshot_path(cluster_id, snapshot_id),
+            options: options
+          )
+        end
+
+        # This is a private alpha release of Cloud Bigtable snapshots. This feature
+        # is not currently available to most Cloud Bigtable customers. This feature
+        # might be changed in backward-incompatible ways and is not recommended for
+        # production use. It is not subject to any SLA or deprecation policy.
+        #
+        # Lists all snapshots associated with the specified cluster.
+        #
+        # @param cluster_id [String]
+        #   The unique name of the cluster for which snapshots should be listed.
+        #   Default value is +'-'+ which list snapshots for all clusters in
+        #   an instance
+        # @param page_size [Integer]
+        #   The maximum number of resources contained in the underlying API
+        #   response. If page streaming is performed per-resource, this
+        #   parameter does not affect the return value. If page streaming is
+        #   performed per-page, this determines the maximum number of
+        #   resources in a page.
+        # @param options [Google::Gax::CallOptions]
+        #   Overrides the default settings for this call, e.g, timeout,
+        #   retries, etc.
+        # @return [Google::Gax::PagedEnumerable<Google::Bigtable::Admin::V2::Snapshot>]
+        #   An enumerable of Google::Bigtable::Admin::V2::Snapshot instances.
+        #   See Google::Gax::PagedEnumerable documentation for other
+        #   operations such as per-page iteration or access to the response
+        #   object.
+        # @raise [Google::Gax::GaxError] if the RPC is aborted.
+        # @example
+        #   require "google/cloud/bigtable"
+        #
+        #   client = Google::Cloud::Bigtable.new(
+        #     client_type: :table,
+        #     instance_id: "instance-id"
+        #   )
+        #
+        #   client.snapshots(cluster_id: "cluster-1").each do |snapshot|
+        #     # Process snapshot
+        #     p snapshot
+        #   end
+        #
+        #   # Or iterate over results one page at a time.
+        #   client.snapshots(cluster_id: "cluster-1").each_page do |page|
+        #     # Process each page at a time.
+        #     page.each do |snapshot|
+        #       # Process snapshot.
+        #       p snapshot
+        #     end
+        #   end
+        #
+        #   # List all snapshots from all clusters
+        #   client.snapshots.each do |snapshot|
+        #     # Process snapshot
+        #     p snapshot
+        #   end
+
+        def snapshots \
+            cluster_id = "-",
```

Comment:
I don't see a test for the default `cluster_id` value of `"="`.

Reply:
I will add.